examples/hello_world/client/node_js/index.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const protoLoader = require('@grpc/proto-loader');
+const grpc = require('@grpc/grpc-js');
+
+const CERT_PATH = __dirname + '/../../../certs/local/ca.pem';
+const PROT_PAH = __dirname + '/../../proto/hello_world.proto';
+
+const packageDefinition = protoLoader.loadSync(PROT_PAH);
```

Comment:
Note: This actually reads the proto file at runtime, as opposed to statically generating js code from the proto file. I went with this because it's relatively straightforward, but happy to switch to using statically generated code. 

For details: https://github.com/protobufjs/protobuf.js#reflection-vs-static-code